README.md

```diff
@@ -18,4 +18,137 @@ A tool to manage your PHP School workshops. Search, install, update & remove wor
+ | PHP7 way       | A PHP School workshop about some new features in | php7way     |     ✘      |
+ |                | PHP7.                                            |             |            |
+ +----------------+--------------------------------------------------+-------------+------------+
+```
+
+### Install
+
+Install a workshop with their package name, generally found from doing a search like above. 
+
+```sh
+ workshop-manager install learnyouphp
+```
+
+You can then get started on your workshop instantly by using it's package name, in this case you would just run `learnyouphp` on the terminal. 
+
+_*Tip:* If an error ever occurs and your not sure what it is, run it with `-vvv` to get more details or create an issue for us to look at_
+
+### Update
+
+A simple way to update a workshop you already have installed. As workshops are just more packages they may include bugs :scream: so keeping them up to date is important!
+
+```sh
+ workshop-manager update learnyouphp
+```
+
+### Uninstall | Remove
+
+Remove a workshop by their package name.
+
+```sh
+ workshop-manager uninstall learnyouphp
+```
+
+### Installed
+
+List the installed workshops, just so you know what you can get working on :wink:
+
+It will also let you know if you need to update any workshops you already have installed.
+
+```sh
+ workshop-manager installed
+ 
+ # Result
+ *** Installed Workshops ***
+ 
+ +----------------+-------------------------------------------------+-------------+---------+------------------------+
+ | Name           | Description                                     | Package     | Version | New version available? |
+ +----------------+-------------------------------------------------+-------------+---------+------------------------+
+ | Learn you PHP! | A simple introduction into PHP and the workshop | learnyouphp | 0.3.1   | Nope!                  |
+ |                | format.                                         |             |         |                        |
+ +----------------+-------------------------------------------------+-------------+---------+------------------------+
+```
+
+### Self-update
+
+Keeping the workshop manager up to date is just as important as updated the workshops themselves. We're not perfect and there are probably bugs in here, or maybe we'll launch some uber cool features, either way you should run this on a regular basis.
+
+```sh
+ workshop-manager self-update
+```
+
+You can then continue using the workshop manager as you was before.
+
+### Rollback
+
+Something go horribly wrong after that `self-update` ? No worries we got your back :wink: just run...
+
+```sh
+ workshop-manager rollback
+```
+
+Then you might want to create an issue for that horrible bug you found :joy:
+
+### Verify
+
+You might need to verify your installation if your running into problems. This command will help diagnose those issues and point you in the right direction. 
```

Comment:
I rekon we should put a note at the top about this, like run this after installing and it will tell you about path stuff and how to set it up.